src/coefficients/MFEMConstantCoefficient.C

```diff
@@ -11,8 +11,9 @@ MFEMConstantCoefficient::validParams()
 }
 
 MFEMConstantCoefficient::MFEMConstantCoefficient(const InputParameters & parameters)
-  : MFEMCoefficient(parameters), coefficient(getParam<double>("value"))
+  : MFEMCoefficient(parameters)
 {
+  coefficient = std::make_shared<mfem::ConstantCoefficient>(getParam<double>("value"));
```

Comment:
Where possible, it's worth keeping this in the list of initialisations rather than in the constructor body